Vagrantfile

```diff
@@ -1,8 +1,38 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+require 'yaml'
+AWS = YAML.load_file 'aws.yml'
+
+puts ""
```

Comment:
Is there a real need for this extra blank line?

Reply:
removed